src/SearchResults.js

```diff
@@ -0,0 +1,36 @@
+import moment from "moment";
+
+const SearchResults = props => (
+    <table className="table table-striped">
+        <thead>
+            <tr className="bg-dark text-white">
+                <th scope="col">ID</th>
+                <th scope="col">Title</th>
+                <th scope="col">First Name</th>
+                <th scope="col">Surname</th>
+                <th scope="col">email</th>
+                <th scope="col">Room ID</th>
+                <th scope="col">Check In Date</th>
+                <th scope="col">Check Out Date</th>
+                <th scope='col'>Number of nights stay</th>
+            </tr>
+        </thead>
+        <tbody>
+            {props.results.map(guest => (
```

Comment:
Don't forget that when you iterate over an array, you'll need to make sure each element has a `key`. Right now there are errors throwing in the browser console because of this.
https://syllabus.codeyourfuture.io/react/week-1/lesson#keys